easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'MakeCp'
+
+name = 'URANOS'
+version = '1.07'
+
+homepage = 'https://github.com/mkoehli/uranos/tree/main'
+description = """Ultra Rapid Adaptable Neutron-Only Simulation for Environmental Research,
+ is a Monte Carlo toolkit specifically tailored for environmental sciences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+buildopts = 'CXXFLAGS="$CXXFLAGS -std=c++17"'
+
+source_urls = [
+    'https://github.com/mkoehli/uranos/tree/main',
+]
+sources = [{
+    'filename': 'uranos-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/mkoehli',
+        'tag': 'v%(version)s',
+        'repo_name': 'uranos',
+    },
+}]
+
+checksums = ['ff90a50e128f257dd0840153c8f6926820f7293ed9eab1c208ec53995ec5b856']
```

Comment:
Checksum won't be stable like this. But we don't need to do this
```
source_urls = ['https://github.com/mkoehli/uranos/releases/download/v%(version)s/']
sources = [{'download_filename': 'URANOS.zip', 'filename': SOURCELOWER_ZIP}]
checksums = ['2d0cdef79f048ad0bcab72a3a9134a16c306ad4b7d25d47ecd875f411c18d310']
```
should work i think

Reply:
You're absolutely right. Okay, thanks, I'll try

Reply:
yikes, the new download path of URANOS doesn't contain a `src` directory just binary releases.

Reply:
Can be resolved. With new [commit](090c7920de204b7f0968d486634c50d1efb7bdf1) it works.